Networking/Networking/Model/StripeAccount.swift

```diff
@@ -62,8 +62,9 @@ public struct StripeAccount: Decodable {
     public init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
         let status = try container.decode(WCPayAccountStatusEnum.self, forKey: .status)
-        let isLiveAccount = try container.decode(Bool.self, forKey: .isLive)
-        let isInTestMode = try container.decode(Bool.self, forKey: .testMode)
+        // TODO. Rollback these two hardcoded values
```

Comment:
This will have to be reverted

Reply:
Let's open an issue so we don't forget :) Or put it in #5733

Reply:
#5921